people_tool/static/people_tool/js/controllers/SearchController.js

```diff
@@ -98,9 +98,23 @@
             // can still display a clickable anchor link
             var lastName = (full.name_last.trim().length > 0) ?
                 full.name_last : '(none)';
-            return '<a href="' + '#/people/' + full.univ_id + '/courses/">'
+            return '<a href="' + '#/people/' + full.univ_id + '/courses/" ' +
+                'ng-click="setSelectedPersonInfo(\''+full.name_last+'\',\''+
+                full.name_first+'\','+full.univ_id+',\''+full.email_address+'\''+',\''+full.role_type_cd+'\')">'
```

Comment:
@eparker71 : so within the setSelectedPersonInfo, the person metadats is being used to build a person object and I do use a service to pass it from one controller to another. [https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/blob/task/sapnamysore/tlt-2073/add_person_details_table/people_tool/static/people_tool/js/services/PersonInfoService.js#L1]